docs/dockerhub-to-mcr-repo-mapping.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Mapping of Docker Hub repositories in /microsoft org to MCR repositories
+description: This articles provides information about the mapping of repositories on Docker Hub owned by the /microsoft org to Microsoft Container Registry (MCR) repositories.
+ms.topic: guidance
+ms.date: 05/18/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Mapping of Docker Hub repositories in `/microsoft` org to MCR repositories
+
+The following table shows the mapping of `/microsoft` org repositories hosted on Docker Hub to repositories hosted in Microsoft Container Registry (MCR). You can use this mapping to migrate your build, deployment, and other automation scripts to reference images and artifacts from Microsoft Container Registry.
```

Comment:
can we make the `/microsoft` text a link to: https://hub.docker.com/r/microsoft/